frontend/public/components/nav/section.tsx

```diff
@@ -95,56 +117,55 @@ export const NavSection = connect(navSectionStateToProps)(
       (section) => plugins.registry.getNavItems(section)
     );
 
+    mapChild = (c: React.ReactElement): React.ReactElement | null => {
```

Comment:
Nit-pick: the `React.ReactElement | null` return type can be automatically inferred, I'd remove it.